Common/src/main/java/com/techarium/techarium/Techarium.java

```diff
@@ -0,0 +1,56 @@
+package com.techarium.techarium;
+
+import com.techarium.techarium.platform.CommonServices;
+import com.techarium.techarium.registry.TechariumBlockEntities;
+import com.techarium.techarium.registry.TechariumBlocks;
+import com.techarium.techarium.registry.TechariumItems;
+import com.techarium.techarium.registry.TechariumMenuTypes;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.Style;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.CreativeModeTab;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.block.Blocks;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Techarium {
+
+	public static final String MOD_ID = "techarium";
+	public static final String MOD_NAME = "Techarium";
+	public static final Logger LOGGER = LogManager.getLogger();
+	public static CreativeModeTab TAB = CommonServices.REGISTRY.registerCreativeTab(new ResourceLocation(MOD_ID, "tab"), () -> new ItemStack(Blocks.DIAMOND_BLOCK));
+	public static final ResourceLocation FONT = Techarium.resourceLocation("techarium");
+	public static final Style STYLE = Style.EMPTY.withFont(FONT);
+
+	public static final boolean DEBUG_MODE = CommonServices.PLATFORM.isDevelopmentEnvironment();  // if we want debug items/blocks
```

Comment:
I don't particularly like the idea of having a debug mode that is enabled in a dev environment. I think it should just be a toggle or something similar.